static/js/people.js

```diff
@@ -825,6 +825,23 @@ export function is_active_user_for_popover(user_id) {
     return false;
 }
 
+export function is_current_user_only_owner() {
+    if (!page_params.is_owner || page_params.is_bot) {
+        return false;
+    }
+
+    let active_owners = 0;
+    for (const person of active_user_dict.values()) {
+        if (person.is_owner && !person.is_bot) {
+            active_owners += 1;
+        }
+        if (active_owners > 1) {
+            return false;
+        }
+    }
+    return true;
```

Comment:
If `active_owners` is not equal to 1 here, then there is an error in our data, because the current user has `page_params.is_owner = true`.

Would it make sense to add a blueslip warning or error here for that unlikely case?

Reply:
It's a reasonable thing to do, certainly. But the most likely bug in such a situation would be a server bug, so I don't think it's important to do so.